src/api/CortexClient.ts

```diff
@@ -91,17 +92,39 @@ export class CortexClient implements CortexApi {
       ? entities.map(entity => applyCustomMappings(entity, customMappings))
       : entities;
 
-    if (gzipContents) {
-      return await this.postWithGzipBody(`/api/backstage/v1/entities/sync`, {
-        entities: withCustomMappings,
-        teamOverrides,
-      });
-    } else {
-      return await this.post(`/api/backstage/v1/entities/sync`, {
-        entities: withCustomMappings,
-        teamOverrides,
-      });
+    const post = async (path: string, body?: any) => {
+        return gzipContents ? await this.postVoidWithGzipBody(path, body) : await this.postVoid(path, body)
+    }
+
+    await this.postVoid('/api/backstage/v2/entities/sync-init')
+
+    for (let customMappingsChunk of chunk(withCustomMappings, CHUNK_SIZE)) {
+      await post(`/api/backstage/v2/entities/sync-chunked`, {
```

Comment:
Are we intentionally doing all of these requests sequentially (vs attempting to parallelize them)?

Reply:
yup it's intentional
us flooding the backend all at once has been causing OOMs which is why we're chunking in the first place